pkg/validationfile/loader.go

```diff
@@ -52,124 +51,96 @@ func PopulateFromFiles(ds datastore.Datastore, filePaths []string) (*PopulatedVa
 // PopulateFromFilesContents populates the given datastore with the namespaces and tuples found in
 // the validation file(s) contents specified.
 func PopulateFromFilesContents(ds datastore.Datastore, filesContents map[string][]byte) (*PopulatedValidationFile, datastore.Revision, error) {
-	var revision datastore.Revision
-	var nsDefs []*core.NamespaceDefinition
+	var schema string
+	var objectDefs []*core.NamespaceDefinition
 	var caveatDefs []*core.CaveatDefinition
-	schema := ""
 	var tuples []*core.RelationTuple
+	var updates []*core.RelationTupleUpdate
+
```

Comment:
```suggestion
```

Reply:
I actually put it there expressly to make it more readable

Reply:
👍🏻